kotlin-app/app/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 29
```

Comment:
I think the compileSdk version should be the latest available version while the minSdk version should be an older version so that it can support multiple sdk, in this project i thought sdk 16 should be old enough

Reply:
Ah okay didn't realize that.

Reply:
Though sdk 16 is WAY too old, lets target level 23 so we have android 6 and up